### ISSUE TITLE
作者您写的ditto代码是不是有点问题呢？

### ISSUE THREAD

**Issue by palem20** [youpengl/FedCAP#1]
下面的这句代码
    self.optimizer_per.step(model.parameters(), self.device)
填入的参数应该是 这样吧？
          self.optimizer_per.step(self.model_per.parameters(), self.device)      
然后还有ditto原文中 不是说损失函数中会加入正则项吗？我看您使用的损失函数中没有使用正则项呢？
                
                out_p = self.model_per(x)
                loss = self.loss(out_p, y)

                self.optimizer_per.zero_grad()
                loss.backward()

![Image](https://github.com/user-attachments/assets/2ef8baec-6340-4134-8609-036042d6941e)

**Comment by youpengl** [comment]
Please refer to https://github.com/youpengl/FedCAP/blob/424fb964cf320266084714a260e284100323bafd/system/flcore/optimizers/fedoptimizer.py#L14z)
